textfilter/vibe/textfilter/markdown.d

```diff
@@ -1091,19 +1091,45 @@ pure @safe {
 private bool parseAutoLink(ref string str, ref string url)
 pure @safe {
 	string pstr = str;
-	if( pstr.length < 3 ) return false;
-	if( pstr[0] != '<' ) return false;
+	if (pstr.length < 3) return false;
+	if (pstr[0] != '<') return false;
 	pstr = pstr[1 .. $];
 	auto cidx = pstr.indexOf('>');
-	if( cidx < 0 ) return false;
+	if (cidx < 0) return false;
 	url = pstr[0 .. cidx];
-	if( anyOf(url, " \t") ) return false;
-	if( !anyOf(url, ":@") ) return false;
```

Comment:
":@" is now valid.

Reply:
I don't think anything changes related to this condition, because it is replicated below as `atidx < 0 && colonidx < 0`. ":@" will not be accepted due to the length check at the top, but indeed this will let a lot of invalid cases go through. True validation has to be done as a separate step